rosbag2_storage/include/rosbag2_storage/storage_interfaces/base_io_interface.hpp

```diff
@@ -47,6 +47,12 @@ class ROSBAG2_STORAGE_PUBLIC BaseIOInterface
    * \returns the size of the bagfile in bytes.
    */
   virtual uint64_t get_bagfile_size() const = 0;
+
+  /**
+   * Returns the identifier for the plugin.
+   * \returns the identifier.
+   */
+  virtual std::string get_identifier() const = 0;
```

Comment:
I'd propose to refine the function name. What do you think about `get_storage_identifier()`? I could see that we could have other things being identified in the future.

Reply:
The reason why I went with `get_identifier` vs `get_storage_identifier` was because I thought it was redundant since the call is on a storage object.

Reply:
yes, I understand that. My thinking though is that identifier or ID could be later used with, e.g. individual instances of the same storage identifier. I think there is no harm in the redundancy.

Reply:
The function actually is first defined in `BaseIOInterface`, should it instead be `get_io_identifier`? Or do you think `get_storage_identifier` would be a better choice?

Reply:
I don't think it'd be super clear to me what an `io` identifier is. Also, we used `storage identifier` also for the command line argument:
https://github.com/ros2/rosbag2/blob/master/ros2bag/ros2bag/verb/play.py#L30

Reply:
That's enough justification on my side to change it to `get_storage_identifier`.